docker/docker-entrypoint.sh

```diff
@@ -108,7 +108,7 @@ sed -i 's/ErrorLog .*/ErrorLog \/dev\/stderr/' /etc/apache2/apache2.conf
 sed -i 's/Timeout 300/Timeout 600/' /etc/apache2/apache2.conf
 
 # Replace actual location of the mapserver depending on the environment                                                                                                                                           
-sed -i 's#MAP_URL_REPLACE#'"$MAP_URL"'#g' /srv/mapserver/topografie.map /srv/mapserver/topografie_wm.map /srv/mapserver/lufo.map /srv/mapserver/infrarood.map                                                     
+sed -i 's#MAP_URL_REPLACE#'"$2"'#g' /srv/mapserver/topografie.map /srv/mapserver/topografie_wm.map /srv/mapserver/lufo.map /srv/mapserver/infrarood.map                                                     
```

Comment:
Dit is vlgs. mij alleen nodig op cvps. Maar, @emacgillavry weet hier het fijne van.

Reply:
Wait, wordt gecorrigeerd in volgende commit.